website/cue/reference/remap/functions/set_semantic_meaning.cue

```diff
@@ -0,0 +1,44 @@
+package metadata
+
+remap: functions: set_semantic_meaning: {
+	category: "Event"
+	description: """
+		Sets a semantic meaning for an event. Note that this function assigns
+		meaning at Vector startup, and has _no_ runtime behavior. It is suggested
+		to put all calls to this function at the beginning of a VRL function. The function
+		cannot be conditionally called (eg: using an if statement cannot stop the meaning
+		from being assigned).
+		"""
+
+	arguments: [
+		{
+			name: "target"
+			description: """
+				The path of the value that will be assigned a meaning.
+				"""
+			required: true
+			type: ["path"]
+		},
+		{
+			name: "meaning"
+			description: """
+				The name of the meaning to assign.
+				"""
+			required: true
+			type: ["path"]
```

Comment:
```suggestion
			type: ["string"]
```
?

Reply:
No, path refers to "queries" in VRL, which are distinct from strings. They aren't quoted like strings.

Reply:
oh this is the meaning :facepalm: yea, you are right.